### DISCUSSION TITLE
How to use below release fix in react-native:0.73.7

### DISCUSSION THREAD

**Discussion by gkasireddy202** [reactwg/react-native-releases#576] [Releases]
I raised this issue: https://github.com/facebook/react-native/issues/44068 in react-native issues. This issue is fixed, and I was asked to pick up the request for 0.73.

https://github.com/reactwg/react-native-releases/issues/520

Could anyone please help me figure out how to fix the issue in react-native:0.73.7?

**Comment by cortinico** [comment]
Please refer to the FAQ doc here:
https://github.com/reactwg/react-native-releases/blob/main/docs/faq.md#what-is-a-pick-request

**Comment by gkasireddy202** [comment]
@cortinico - Thanks for your help.
Can you please help me how do I know if my fix/feature is in a certain release(0.73.7)?

My issue: https://github.com/facebook/react-native/issues/44068

Pick request: https://github.com/reactwg/react-native-releases/issues/520

**Reply by cortinico** [reply]
Your fix is in 0.73 as per your pick request: https://github.com/reactwg/react-native-releases/issues/520

It's included in 0.73.10, you can see it in the release branch:
https://github.com/facebook/react-native/commits/0.73-stable/

**Comment by gkasireddy202** [comment]
@cortinico  - How to fix this issue on react-native:0.73.7?

**Reply by cortinico** [reply]
> @cortinico - How to fix this issue on react-native:0.73.7?

You need to update to 0.73.10 to have this fix included

**Comment by gkasireddy202** [comment]
@cortinico  - There is any chance to fix this issue on 0.73.7. we recently upgraded my project to 0.73.7.
My client does not approve of upgrading the project again.

**Reply by cortinico** [reply]
0.73.7 has been already released. You'll need to update to 0.73.10 (or a later version) to have this fix

**Reply by gkasireddy202** [reply]
Thanks for your update.
I will update it to 0.73.10.
Can you please look at this issue:https://github.com/facebook/react-native/issues/44612
